src/styles/hc-2018/ternary.styl

```diff
@@ -173,7 +173,13 @@ h5, h5 span {
     font-weight: 300;
     line-height: 120%;
 }
-p, li, a:not(h5 a) {
```

Comment:
Does this change have an effect on the styling?

Reply:
Yep. Believe it or not, Chrome's user agent stylesheet has more specific selectors than Safari's, so you sometimes have to break up rules for Chrome.

Reply:
That doesn't sound like a particularly discoverable bug!  Could we fix this by adding some normalisation rules (e.g. [normalize.css](https://necolas.github.io/normalize.css/)) before our main CSS rules?

Reply:
In principle I would agree, but given that a lot of styling is already done, I'm not sure many more cases like these will occur. So I don't really mind either way!